posthog/api/event_definition.py

```diff
@@ -135,8 +125,8 @@ def _ordering_params_from_request(
             else:
                 order_direction = "ASC"
         else:
-            order = "volume_30_day"
-            order_direction = "DESC"
+            order = "name"
```

Comment:
maybe last_seen_at is a more useful default ordering 🤷

Reply:
probably, but then I'd have to add it as a column to the table... and that's a new development :D